backend/api/v1/endpoints/connections.py

```
@@ -1,32 +1,43 @@
 import logging
-from fastapi import APIRouter, Depends, HTTPException
+from fastapi import APIRouter, Depends, HTTPException, Query
```

Ruff (0.8.2)

`fastapi.Query` imported but unused

Remove unused import: `fastapi.Query`

(F401)